config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -456,8 +456,8 @@ presubmits:
         default_repo: kubernetes
         path_strategy: legacy
       gcs_credentials_secret: service-account
-      grace_period: 15000000000
-      timeout: 2400000000000
+      grace_period: 15s
+      timeout: 40m0s
```

Comment:
nit: do we need the 0s here? That's odd

Reply:
Unfortunately that is how duration strings are printed: https://play.golang.org/p/xyrKD-34gfP